app/app/context.py

```diff
@@ -108,35 +121,22 @@ def preprocess(request):
         if record_join:
             Activity.objects.create(profile=profile, activity_type='joined')
 
+        chat_access_token = profile.gitcoin_chat_access_token
+        chat_id = profile.chat_id
     # handles marketing callbacks
     if request.GET.get('cb'):
         callback = request.GET.get('cb')
         handle_marketing_callback(callback, request)
 
-    chat_unread_messages = False
-
-    if profile and hasattr(profile, 'chat_id'):
```

Comment:
hey @androolloyd does the new chat system replace this functionality somewhere?

should i see a notification somewhere? https://bits.owocki.com/4gumPZgN

Reply:
i see the changes in base.js but its not working in prod rn